patches/core/ungoogled-chromium/block-trk-and-subdomains.patch

```diff
@@ -142,46 +142,6 @@
        "url_request/url_fetcher.cc",
        "url_request/url_fetcher.h",
        "url_request/url_fetcher_core.cc",
---- a/android_webview/browser/net/aw_url_request_context_getter.cc
-+++ b/android_webview/browser/net/aw_url_request_context_getter.cc
-@@ -62,6 +62,7 @@
- #include "net/ssl/ssl_config_service.h"
- #include "net/url_request/data_protocol_handler.h"
- #include "net/url_request/file_protocol_handler.h"
-+#include "net/url_request/trk_protocol_handler.h"
- #include "net/url_request/url_request_context.h"
- #include "net/url_request/url_request_context_builder.h"
- #include "net/url_request/url_request_intercepting_job_factory.h"
-@@ -132,6 +133,9 @@ std::unique_ptr<net::URLRequestJobFactor
-       url::kDataScheme, std::make_unique<net::DataProtocolHandler>());
-   DCHECK(set_protocol);
-   set_protocol = aw_job_factory->SetProtocolHandler(
-+      url::kTraceScheme, std::make_unique<net::TrkProtocolHandler>());
-+  DCHECK(set_protocol);
-+  set_protocol = aw_job_factory->SetProtocolHandler(
-       url::kBlobScheme,
-       base::WrapUnique((*protocol_handlers)[url::kBlobScheme].release()));
-   DCHECK(set_protocol);
---- a/chromecast/browser/url_request_context_factory.cc
```

Comment:
Are you confident that removing the changes to `chromecast/browser/url_request_context_factory.cc` will not adversely effect the functionality of this patch?

Reply:
I am not confident in that, and I'm still trying to figure out how to handle that.

Reply:
I have managed to fix this, but there's nothing I can do for `android_webview/browser/net/aw_url_request_context_getter.cc` because that file, along with that specific directory, does not exist.